docs/organizer-guide/meetups/livestreaming-meetups.rst

```diff
@@ -0,0 +1,40 @@
+Livestreaming or Recording a Meetup
+===================================
+
+Whether due to health or environmental reasons or because you live somewhere where people are spread out over a large area, maybe you want to try live streaming a meetup.
+
+We welcome streams or uploads to the Podcast YouTube channel, but first, we have to add you to the channel. A few days ahead of time, message @chrisward @tomjohnson or @jaredmorgs or the #podcast Slack channel with your email address and we will add you to the channel.
+
+With that access, you can upload a recorded meetup or live stream to the channel.
+
+Recording a Meetup
+------------------
+
+To record a meetup, all you need is a microphone. An external microphone is best, but the internal microphone on your laptop can work if you have nothing else.
+
+If you want video too, then again, an external camera is best, but positioned well, the internal camera in a laptop can work.
+
+If you want to mix the speaker and slides onto one recording, you can do this after the recording with video editing software, or live using software such as `Open Broadcast Studio (OBS) <https://obsproject.com/>`_.
+
+Once the recording is complete, you need to convert it to `a format YouTube supports <https://support.google.com/youtube/troubleshooter/2888402?hl=en>`_.
+
+Uploading a Recorded Meetup
```

Comment:
Maybe we should add that if people use music or video clips in their presentations, they need to make sure they use copyright-free or original sound, to prevent the YouTube overlords from flagging copyrighted content